providers/src/airflow/providers/google/cloud/operators/alloy_db.py

```diff
@@ -206,13 +207,40 @@ def __init__(
         self.cluster_configuration = cluster_configuration
         self.is_secondary = is_secondary
 
-    def execute(self, context: Context) -> Any:
-        message = (
-            "Validating a Create AlloyDB cluster request."
-            if self.validate_request
-            else "Creating an AlloyDB cluster."
+    def _get_cluster(self) -> proto.Message | None:
+        self.log.info("Checking if the cluster %s exists already...", self.cluster_id)
+        try:
+            cluster = self.hook.get_cluster(
+                cluster_id=self.cluster_id,
+                location=self.location,
+                project_id=self.project_id,
+            )
+        except NotFound:
+            self.log.info("The cluster %s does not exist yet.", self.cluster_id)
+        except Exception as ex:
+            raise AirflowException(ex) from ex
+        else:
+            self.log.info("AlloyDB cluster %s already exists.", self.cluster_id)
+            result = alloydb_v1.Cluster.to_dict(cluster)
+            return result
+        return None
+
+    def execute(self, context: Context) -> proto.Message:
```

Comment:
Regarding the return value, expected types are dict | None?

Reply:
Please check return values in operators file, elsewhere `proto.Message` suggested. 
I think it should be a dict at most cases or at least more specific proto type, like `Instance`, `Cluster` etc.

Reply:
Updated. Thank you!